addons/mail/controllers/discuss/public_page.py

```diff
@@ -10,9 +10,9 @@
 from odoo.http import request
 from odoo.tools import consteq
 from odoo.tools.misc import get_lang
+from .public_controller import PublicController
 
-
-class PublicPageController(http.Controller):
+class PublicPageController(PublicController):
```

Comment:
not particularly fan of complex inheritance chain

Reply:
would you rather having the `add_current_persona_to_channel` function as a standalone helper?